src/progress.rs

```diff
@@ -62,6 +69,23 @@ impl Drop for Progress<'_> {
     }
 }
 
+fn draw_progress(progress: f32, buffer: &mut String, width: usize) {
+    const CHARS: [char; 9] = [' ', '▏', '▎', '▍', '▌', '▋', '▊', '▉', '█'];
```

Comment:
The last one seems to be unused, right?

Reply:
Refactored to pull everything from this array.